src/test/script_tests.cpp

```diff
@@ -1704,11 +1747,23 @@ static void AssetTest(const UniValue& test)
         PrecomputedTransactionData txdata;
         txdata.Init(tx, std::vector<CTxOut>(prevouts));
         CachingTransactionSignatureChecker txcheck(&tx, idx, prevouts[idx].nValue, true, txdata);
+
+        #if defined(HAVE_CONSENSUS_LIB)
+        CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
+        stream << tx;
+        CDataStream spent_outputs_stream(SER_NETWORK, PROTOCOL_VERSION);
+        spent_outputs_stream << prevouts;
+        #endif
+
         for (const auto flags : ALL_CONSENSUS_FLAGS) {
             // If a test is supposed to fail with test_flags, it should also fail with any superset thereof.
             if ((flags & test_flags) == test_flags) {
                 bool ret = VerifyScript(tx.vin[idx].scriptSig, prevouts[idx].scriptPubKey, &tx.vin[idx].scriptWitness, flags, txcheck, nullptr);
                 BOOST_CHECK(!ret);
+                #if defined(HAVE_CONSENSUS_LIB)
```

Comment:
Generally macro statements aren't indented like the code they're included in.

Reply:
Thanks, didn't know that. Fixed in d82e5de6a41794cc8fd1e3afaa12af26bb7be926